### PR TITLE
[Konvoy] Adding step to download containerd bundle for airgapped builds

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-90568

## Description of changes being made

Added some steps to AMI creation.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4558.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md [hunk 1]
```diff
@@ -61,6 +61,26 @@ Using the [Konvoy Image Builder](../../../../image-builder), you can build an AM
...
+    ```
+Available packages are:
+
+-   `centos-7.9-x86_64`
+-   `rhel-7.9-x86_64`
+-   `rhel-8.2-x86_64`
+-   `ubuntu-20.04-x86_64`
+-   `ubuntu-18.04-x86_64`
+
+1. Download the the `containerd` bundle.
```
Comment (cmastr) [inline pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md:79]:
```suggestion
1. Download the `containerd` bundle.
```

**Review by cmastr — APPROVED** [review]

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/create-an-ami/index.md [hunk 2]
```diff
@@ -61,12 +61,32 @@ Using the [Konvoy Image Builder](../../../../image-builder), you can build an AM
     ```bash
     curl --output artifacts/pip-packages.tar.gz --location https://downloads.d2iq.com/dkp/airgapped/pip-packages/pip-packages.tar.gz
     ```
+NOTE: If you're installing on v2.2.2 or above, you will need to continue onto step 7.
+
+1. Set your `containerd` operating system:
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/create-an-ami/index.md:66]:
```suggestion
1. Set a variable that indicates which os-specific `containerd` bundle to download:
2. ```

File: pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md [hunk 3]
```diff
@@ -61,6 +61,26 @@ Using the [Konvoy Image Builder](../../../../image-builder), you can build an AM
     ```bash
     curl --output artifacts/pip-packages.tar.gz --location https://downloads.d2iq.com/dkp/airgapped/pip-packages/pip-packages.tar.gz
     ```
+    <p class="message--note"><strong>NOTE: If you're installing on DKP v2.2.2 or above, you will need to continue onto step 7.</p>
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md:64]: This note is not needed on 2.3; this is just a part of the steps now.

```suggestion
```

File: pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md [hunk 4]
```diff
@@ -61,6 +61,26 @@ Using the [Konvoy Image Builder](../../../../image-builder), you can build an AM
...
+
+1. Set your `containerd` operating system:
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md:66]: Please apply the same suggestions I made above to this page as well.

**Review by cneth — CHANGES_REQUESTED** [review]
AFAIK, similar steps will need to be added to other places as well, this is just for AWS, I believe that both vSphere and the preprovisioned airgapped (dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/prerequisites-airgapped/) pages will need adjustments.

**Comment by mwrightd2** [comment]
@cneth I'm creating a separate PR for the other environments since this JIRA ticket is purely for AWS. Since that's the case would you be ok with approving?

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/create-an-ami/index.md [hunk 5]
```diff
@@ -61,12 +61,34 @@ Using the [Konvoy Image Builder](../../../../image-builder), you can build an AM
     ```bash
     curl --output artifacts/pip-packages.tar.gz --location https://downloads.d2iq.com/dkp/airgapped/pip-packages/pip-packages.tar.gz
     ```
+     NOTE: If you're installing V2.2.0 or V2.2.1, skip to step 9, as those releases did not have separate `containerd` packages.
+
+1. Set a variable that indicates which os-specific `containerd` bundle to download:
+2. ```
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/create-an-ami/index.md:67]:
```suggestion
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/create-an-ami/index.md [hunk 6]
```diff
@@ -61,12 +61,33 @@ Using the [Konvoy Image Builder](../../../../image-builder), you can build an AM
...
+    ```bash
+    export CONTAINERD_OS=centos-7.9-x86_64
+    ```
+Note: available packages are:
+
+-   `centos-7.9-x86_64`
+-   `rhel-7.9-x86_64`
+-   `rhel-8.2-x86_64`
+-   `ubuntu-20.04-x86_64`
+-   `ubuntu-18.04-x86_64`
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/create-an-ami/index.md:77]:
```suggestion
    Note: available packages are:

    -   `centos-7.9-x86_64`
    -   `rhel-7.9-x86_64`
    -   `rhel-8.2-x86_64`
    -   `ubuntu-20.04-x86_64`
    -   `ubuntu-18.04-x86_64`
```

File: pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md [hunk 7]
```diff
@@ -62,6 +62,25 @@ Using the [Konvoy Image Builder](../../../../image-builder), you can build an AM
...
+    ```bash
+    export CONTAINERD_OS=centos-7.9-x86_64
+    ```
+Available packages are:
+
+-   `centos-7.9-x86_64`
+-   `rhel-7.9-x86_64`
+-   `rhel-8.2-x86_64`
+-   `ubuntu-20.04-x86_64`
+-   `ubuntu-18.04-x86_64`
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/aws/air-gapped/create-an-ami/index.md:76]:
```suggestion
    Available packages are:

    -   `centos-7.9-x86_64`
    -   `rhel-7.9-x86_64`
    -   `rhel-8.2-x86_64`
    -   `ubuntu-20.04-x86_64`
    -   `ubuntu-18.04-x86_64`
```

**Review by cneth — APPROVED** [review]

**Pull request merged**